src/entities/account.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, PrimaryKey, ManyToMany, Collection, OneToMany } from "@mikro-orm/core";
+import { v4 } from "uuid";
+import { AccountUser } from "./account-user.entity";
+import { Project } from "./project.entity";
+import { User } from "./user.entity";
+
+@Entity()
+export class Account {
+  @PrimaryKey({ unique: true })
+  id: string = v4();
+
+  // An Org has users so owner
+  @ManyToMany({
+    entity: () => User,
+    pivotEntity: () => AccountUser,
+    type: User,
```

Comment:
this is also not needed/ignored, for relations decorators its about the `entity` option, `type` is ignored

Reply:
@B4nan Noted, thank you!